test/specs/configs/full.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+# defines the default content repository
+# content: local
+content: https://github.com/adobe/project-helix.io/blob/master
+
+# defines the default code repository
+code: https://github.com/adobe/project-helix.io/blob/master
+
+# defines the default static root relative to the code repository
+staticRoot: /webroot
+
+# defines the directory index file
+directoryIndex: index.html
+
+# define additional configs (strains)
+strains:
+  default:
```

Comment:
This is a breaking change. Right now, we have

```yaml
- strain
  name: default
```

I like your approach better, as it is closer to YAML semantics.

Reply:
ok